### ISSUE TITLE
Float support

### ISSUE THREAD

**Issue by sdogruyol** [waterlink/active_record.cr#38]
Currently `Float` is missing and its pretty much one of the most common types in db

**Comment by waterlink** [comment]
You are right. This time I will increase unit-test coverage of types themselves separately, and then reduce duplication via refactoring, using nice generation macro, ideally making definition of new type as simple as:

``` crystal
ActiveRecord.register_type(:struct, Float32, default: 0.0, comparable: true)
ActiveRecord.register_type(:struct, Float64, default: 0.0, comparable: true)

ActiveRecord.register_field_types(FloatTypes, Float32|Float64)
```

**Comment by waterlink** [comment]
I have released version `0.2.0` with `Floats` support
